lib/far_mar/market.rb

```diff
@@ -0,0 +1,38 @@
+class FarMar::Market < FarMar::Base
+  attr_accessor :id, :name, :address, :city, :county, :state, :zip
+  FILE = ("support/markets.csv")
+
+  def initialize(attrs)
+   @id= attrs[0].to_i
```

Comment:
Just for style, you should have spaces before and after all `=`, so `@id = attrs[0].to_i`. And even further it's nice to line up the equal signs vertically just to make it readable.